rhamt/entities/analysis_configuration.py

```diff
@@ -32,8 +35,9 @@ def is_displayed(self):
 class AnalysisConfiguration(Updateable, NavigatableMixin):
     """Analysis Configuration"""
 
-    def __init__(self, application):
+    def __init__(self, application, project_name):
         self.application = application
+        self.project_name = project_name
```

Comment:
Now, we have `project_name` as new attribute.
is it ok to move it into collection?

Reply:
There will never be collection of analysis configuration . We have to select  a particular project first and then edit it's configuration so it will be always one configurtaion . That's why I didn't create collection . Also we cannot create configuration so there will not be any methods under collection may be we can just put instantiate . If you think that's the better way then I can change it

Reply:
sure thing